client/src/common.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Header {
+    // TODO
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct BlockFinalizationProof {
+    // TODO
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct MerkleProof {
+    // TODO
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct FungibleTokenTransfer {
+    pub token_id: String,
```

Comment:
I think token_address would be better than token_id.

Reply:
fixed